### PR TITLE
Add goimports beautifier for Go language

### PR DESCRIPTION
**Pull request by tanyadegurechaff**
### What does this implement/fix? Explain your changes.
This adds goimports beautifier for Go language.

### Does this close any currently open issues?

...

### Any other comments?

...

### Checklist

Check all those that are applicable and complete.

- [x] Merged with latest `master` branch
- [x] Regenerate documentation with `npm run docs`
- [x] Add change details to `CHANGELOG.md` under "Next" section
- [ ] Added examples for testing to [examples/ directory](examples/)
- [x] Travis CI passes (Mac support)
- [x] AppVeyor passes (Windows support)

### CODE REVIEW

File: src/beautifiers/goimports.coffee [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+"use strict"
+Beautifier = require('./beautifier')
+
+module.exports = class Goimports extends Beautifier
+  name: "goimports"
+  link: "https://godoc.org/golang.org/x/tools/cmd/goimports"
+  isPreInstalled: false
+
+  options: {
+    Go: true
```
Comment (Glavin001) [inline src/beautifiers/goimports.coffee]: Looks like no options are supported, therefore this should be `false`. 😃

File: src/beautifiers/goimports.coffee [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+Requires https://godoc.org/golang.org/x/tools/cmd/goimports
+###
+
+"use strict"
+Beautifier = require('./beautifier')
+
+module.exports = class Goimports extends Beautifier
+  name: "goimports"
+  link: "https://godoc.org/golang.org/x/tools/cmd/goimports"
+  isPreInstalled: false
```
Comment (Glavin001) [inline src/beautifiers/goimports.coffee]: This has been deprecated.

We have switched to using `Executables`: https://github.com/Glavin001/atom-beautify/issues/1687

See https://github.com/Glavin001/atom-beautify/blob/master/src/beautifiers/sass-convert.coffee#L7-L20 and https://github.com/Glavin001/atom-beautify/blob/master/src/beautifiers/sass-convert.coffee#L30 for simple example. And another: example https://github.com/Glavin001/atom-beautify/blob/master/src/beautifiers/autopep8.coffee#L12-L56

Please let me know if you have any questions! This is a new process to help standardize creating non-preinstalled beautifiers. Thanks!

Comment (tanyadegurechaff) [inline src/beautifiers/goimports.coffee]: It seems that goimports doesn't have version option or subcommand.

Comment (Glavin001) [inline src/beautifiers/goimports.coffee]: If `--version` is not available then I've used `--help` and returned a fake version of `0.0.0` like https://github.com/Glavin001/atom-beautify/blob/master/src/beautifiers/beautysh.coffee#L14-L16

**Comment by tanyadegurechaff** [comment]
Could you re-generate the documentation and commit to master?

**Comment by Glavin001** [comment]
> Could you re-generate the documentation and commit to master?

Are you asking me? If so, I can, however I highly encourage you to do this part yourself. I need to see  that your changes were setup properly before I merge and the auto-generated documentation usually shows it all worked.

It's easy: `npm run docs` 😄 
Are you having troubles with this step? Any errors?

**Comment by tanyadegurechaff** [comment]
I don't want to include changes related to #1708

**Comment by Glavin001** [comment]
Oh, I see what you mean. Pushed!

**Comment by tanyadegurechaff** [comment]
Thank you for your help!

File: src/beautifiers/goimports.coffee [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  executables: [
+    {
+      name: "goimports"
+      cmd: "goimports"
+      homepage: "https://godoc.org/golang.org/x/tools/cmd/goimports"
+      installation: "https://godoc.org/golang.org/x/tools/cmd/goimports"
+      version: {
+        # Does not display version
+        args: ['--help'],
+        parse: (text) -> text.indexOf("usage: goimports") isnt -1 and "0.0.0",
```
Comment (Glavin001) [inline src/beautifiers/goimports.coffee]: I also created a Docker image for `goimports`: https://hub.docker.com/r/unibeautify/goimports/
I am going to use it during my testing right now.

**Comment by Glavin001** [comment]
Worked great! Thank you for contributing!

**Comment by Glavin001** [comment]
Published to v0.30.3

**Pull request merged**